.papr.yml

```diff
@@ -70,6 +70,8 @@ inherit: true
 context: f25-experimental-api
 env:
   CONFIGOPTS: '--enable-experimental-api'
+  CFLAGS: '-Werror=unused-variable'
```

Comment:
Well, how about we just turn on `-Werror` for ci?   Going back to [my original philosophy on compiler warnings](https://mail.gnome.org/archives/desktop-devel-list/2012-July/msg00100.html) - as (a former) gnome-continuous CI administrator, owning the build of *hundreds* of separate projects, some of which didn't (and still don't) have any CI at all, `-Werror` is awful.

But we're in the inverse situation, we have a fixed tool chain for *one* project.